### PR TITLE
Less pushy

### PR DESCRIPTION
**Pull request by calvin-brizzi**
Based on pull reques #3, by @Vulcanit3

I think part of the knee-jerk response is the bot suggesting what people meant, the response here is repetitive, but explains the issue, provides a suggestion that people can't really complain about (we've had people saying "gang" is inappropriate) and gives a link.

### CODE REVIEW

File: slackbot-guysgals-reasoning.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Why the Auto Response?
+
+The ZATech community has some people who feel excluded by certain words. In particular there was a point raised in #diversity about the word `guys`.
```
Comment (zoidyzoidzoid) [inline slackbot-guysgals-reasoning.md]: I don't like this first sentence. I feel like it subtly puts blame on the people feeling excluded and makes it sound like the portion of people that feel excluded by the term, within the community or not, is smaller than it is.

File: slackbot-guysgals-reasoning.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+That's not the MAIN reason why the slackbot auto response exists though. There's a bigger purpose. The purpose is to try and get people to be more aware about the language they use and to try and be more inclusive towards other humans. Regardless of Class, Race, Gender, Sex or other divides that humans love putting up.
+
+We want to be a loving and inclusive community, and we know that the community is made up of the individuals. Thus if the individuals are made more aware of how better to be inclusive to the group, the community as a whole becomes more inclusive.
+
+
+# What we're not trying to do
+
+This is NOT about policing language or stopping freedom of speech.
+It's about getting people to be a little bit more aware of the language they use and the implications thereof.
```
Comment (zoidyzoidzoid) [inline slackbot-guysgals-reasoning.md]: Nitpick: This could just say `their implications` at the end.

File: slackbot-guysgals-reasoning.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+# I don't agree with this, what now?
+
+Please don't feel excluded or shamed, there's been a huge amount of discussion on several channels about diversity, this slackbot and general inclusivity.
+If you want to change something about how it works, be that a copy change or anything, we really do welcome suggestions to refine the bot and are very open about it's flaws, limitations and aggravations. It is slightly limited at the moment (it's not a custom bot, simply an auto-responder).
```
Comment (zoidyzoidzoid) [inline slackbot-guysgals-reasoning.md]: *its flaws.

File: slackbot-guysgals-reasoning.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+Some relevant channels to look at include:
+* `#meta-guys-response` for talking about this specific slackbot
+* `#diversity` - Discussion of ways and means of ensuring that groups of people that are under-represented in tech (gender, ethnicity, sexual orientation, ability, and class) do not feel excluded and are given every possible opportunity to be part of the community.
+* `#others` - Stories, advice and sympathy on being an "other" in tech
+* `#codeofconduct` - Chatting about the ZATech CoC - https://github.com/zatech/code-of-conduct
```
Comment (zoidyzoidzoid) [inline slackbot-guysgals-reasoning.md]: `ZATech [Code of Conduct](https://github.com/zatech/code-of-conduct)`

File: slackbot-guysgals-reasoning.md (hunk 1 above)
Comment (calvin-brizzi) [reply slackbot-guysgals-reasoning.md]: Ah, I see what you mean and I agree. I'll switch it around, more along the lines of "Certain words can be alienating to various members of the ZATech community, in particular...."

As discussed elsewhere a small paragraph on "even if you have no intention of being alienating a lot of people who aren't men find it othering etc etc"

File: slackbot-guysgals-reasoning.md (hunk 2 above)
Comment (calvin-brizzi) [reply slackbot-guysgals-reasoning.md]: "It's about getting people to be a little bit more aware of the language they use and their implications." doesn't sound right. I think I'm misunderstanding what you're trying to say :)

File: slackbot-guysgals-reasoning.md (hunk 1 above)
Comment (JohnRoux) [reply slackbot-guysgals-reasoning.md]: I don't think it's necessary or useful to say "a lot of people who aren't men".
Just saying "a lot of people" sounds fine.

File: slackbot-guysgals-reasoning.md (hunk 2 above)
Comment (JohnRoux) [reply slackbot-guysgals-reasoning.md]: How about:
It's about getting people to be a little bit more aware of the language they use and the implications of that language.

I agree that just ending off with 'their implications' sounds weird, it also doesn't tie it to the language used

File: slackbot-guysgals-reasoning.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+* `#asktheadmins` - Want to ask the admins a question but not sure who they are or what's going on, this is the place
+
+# Some frequently asked questions 
+
+* **Isn't it gender neutral in its plural form?** Maybe in certain situations, but which situations those are is ambiguous, [here's an unscientific survey](https://jvns.ca/blog/2013/12/27/guys-guys-guys/) that shows that not everyone agrees if you find a better survey, please post it for discussion in the groups above. What is for certain is that there are members of our community that do not think it is gender neutral.
```
Comment (tigrrrly) [inline slackbot-guysgals-reasoning.md]: break into new sentence: 
"...that shows that not everyone agrees. If you find a better..."

**Review by n1c — APPROVED** [review]

**Review by ChristianWitts — APPROVED** [review]
LGTM 👍

**Pull request merged**